### ISSUE TITLE
error importing pygrass Mapset

### ISSUE THREAD

**Issue by lucadelu** [zarch/grass-session#3]
```
import grass_session as gs
gs.Session(grassbin=gs.GRASSBIN,gisdb='/mnt/gluster1/grassdata', location='latlong', mapset='cosmorea6_t2m')
from grass.pygrass.gis import Mapset
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-9-3e7c8394a115> in <module>()
----> 1 from grass.pygrass.gis import Mapset

/home/lucadelu/compilati/grass_trunk/dist.x86_64-pc-linux-gnu/etc/python/grass/pygrass/gis/__init__.py in <module>()
     11 
     12 
---> 13 import grass.lib.gis as libgis
     14 from grass.pygrass.errors import GrassError
     15 from grass.script.utils import encode, decode

/home/lucadelu/compilati/grass_trunk/dist.x86_64-pc-linux-gnu/etc/python/grass/lib/gis.py in <module>()
     21 # Begin libraries
     22 
---> 23 _libs["grass_gis.7.5.svn"] = load_library("grass_gis.7.5.svn")
     24 
     25 # 1 libraries

/home/lucadelu/compilati/grass_trunk/dist.x86_64-pc-linux-gnu/etc/python/grass/lib/ctypes_loader.pyc in load_library(self, libname)
     60         for path in paths:
     61             if os.path.exists(path):
---> 62                 return self.load(path)
     63 
     64         raise ImportError("%s not found." % libname)

/home/lucadelu/compilati/grass_trunk/dist.x86_64-pc-linux-gnu/etc/python/grass/lib/ctypes_loader.pyc in load(self, path)
     76                 return ctypes.cdll.LoadLibrary(path)
     77         except OSError as e:
---> 78             raise ImportError(e)
     79 
     80     def getpaths(self, libname):

ImportError: libgrass_datetime.7.5.svn.so: cannot open shared object file: No such file or directory
```

All the pygrass modules seems to be broken

The environment seems correct... inside GRASS GIS the pygrass modules work properly

**Comment by zarch** [comment]
From my understanding, this problem is due to the fact that is not possible to import dynamically the library path into an existing process. So even if the `grass-session` is setting the correct path in the `LD_LIBRARY_PATH` variable, since this variable change is not valid for the current process we can not use all the libraries that are using ctypes.

A work around is to set the `LD_LIBRARY_PATH` before calling the python file. You can set the `LD_LIBRARY_PATH` through the command line or set the variable into your `~/.bashrc` file.
For example if I define a python file such as:

```python
# file: test.py
import grass_session
from grass.pygrass.vector import VectorTopo

print('DONE!')
```
You can execute the program with:

```bash
$ LD_LIBRARY_PATH=/home/pietro/.local/grass-7.5.svn/lib \
   GRASSBIN=/home/pietro/.local/bin/grass75 \
   python test.py
GRASSBIN: /home/pietro/.local/bin/grass75
GISBASE: /home/pietro/.local/grass-7.5.svn
DONE!
```
These has the disadvantage that you need to know in advance where you libraries are...

**Comment by lrntct** [comment]
Thank you @zarch for your work. I tried grass-session and encountered the same issue.
This ctypes libraries bootstrapping problem in GRASS is a pita really.
In itzi I solved it by setting LD_LIBRARY_PATH (or equivalent on other systems) and then re-executing the program. It feels a bit like an ugly hack, but it works. See:
https://bitbucket.org/itzi-model/itzi/src/master/itzi/itzi.py
We had a discussion on the ML back then:
http://lists.osgeo.org/pipermail/grass-dev/2016-September/082379.html

If you think that this could be an acceptable solution for grass-session, I can work on a PR.
